vcd/datasource_vcd_edgegateway.go

```diff
@@ -97,6 +107,90 @@ func datasourceVcdEdgeGateway() *schema.Resource {
 				Computed:    true,
 				Description: "'accept' or 'deny'. Default 'deny'",
 			},
+			"fips_mode_enabled": &schema.Schema{
+				Type:        schema.TypeBool,
+				Computed:    true,
+				Description: "Enable FIPS mode. FIPS mode turns on the cipher suites that comply with FIPS. (False by default)",
+			},
+			"use_default_route_for_dns_relay": &schema.Schema{
+				Type:        schema.TypeBool,
+				Computed:    true,
+				Description: "If true, default gateway will be used for the edge gateways' default routing and DNS forwarding.(False by default)",
+			},
+			"external_network": {
+				Type:        schema.TypeSet,
```

Comment:
Please confirm whether using `TypeSet` leads to the issue where changing (updating) value of any field in HCL shows the whole `external_network` block as being deleted and re-created during a `terraform plan`.

Reply:
Yes, this will show it. Something like that:

![image](https://user-images.githubusercontent.com/15804230/69730986-b7074c80-1131-11ea-9a8d-bfa1f8fcc282.png)

Reply:
A remotely related issue at HashiCorp for reference:
https://github.com/hashicorp/terraform/issues/21901